{{ cookiecutter.app_name }}/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:trusty
+MAINTAINER ODL DevOps <mitx-devops@mit.edu>
+
+
+# Add package files
+ADD requirements.txt /tmp/requirements.txt
+ADD test_requirements.txt /tmp/test_requirements.txt
+ADD apt.txt /tmp/apt.txt
+WORKDIR /tmp
+
+# Install base packages
+RUN apt-get update
+RUN apt-get install -y $(grep -vE "^\s*#" apt.txt  | tr "\n" " ")
+RUN pip install pip --upgrade
+
+# Install project packages
+RUN pip install -r requirements.txt
+RUN pip install -r test_requirements.txt
+
+# Add, and run as, non-root user.
+RUN mkdir /src
+RUN adduser --disabled-password --gecos "" --home=/src --no-create-home mitodl
+RUN chown -R mitodl:mitodl /src
+
+# Add project
+ADD . /src
```

Comment:
good catch, that order was swapped around like that when I was trying to figure out the pip problem